euphony/src/main/java/co/euphony/rx/EuRxManager.java

```diff
@@ -81,6 +85,13 @@ private boolean listenOnJava() {
 					return false;
 			}
 			mListenThread.start();
+			if (timeout > 0) {
+				EuTimer euTimer = new EuTimer(mListenThread, () -> {
+					listener.onTimeOut();
```

Comment:
if you substitute `empty lambda function` for unnecessary function call, listener's null check should be here.
```suggestion
					if(listener != null)
						listener.onTimeOut();
```